src/replication.c

```diff
@@ -1944,7 +1944,7 @@ void replicationSetMaster(char *ip, int port) {
     disconnectAllBlockedClients(); /* Clients blocked in master, now slave. */
 
     /* Force our slaves to resync with us as well. They may hopefully be able
-     * to partially resync with us, but we can notify the replid change. */
+     * to partially resync with us, but we can notify the replied change. */
```

Comment:
```suggestion
     * to partially resync with us, but we can notify the replid change. */
```
not everything that's not in the dictionary is a typo!